src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -298,10 +310,21 @@ export default {
 				location
 			);
 		},
+		// We will need to factor in categories at some point
```

Comment:
Not sure what this comment means. Consider adding a ticket and referencing here, if future changes are needed.